sklearn_numba_dpex/kmeans/drivers.py

```diff
@@ -663,4 +746,4 @@ def kmeans_plusplus(
         centers_t[:, c] = X_t[:, center_index]
         center_indices[c] = center_index
 
-    return centers_t.T, center_indices
+    return centers_t, center_indices
```

Comment:
What is the reason for this change (i.e. not transposing anymore)?

Reply:
"drivers" in drivers.py file now take and return the raw-est input and outputs. Pre and post processing regarding transposing, type, dtype, etc, that is required upstream or downstream is now located in the engine. I think those are better concepts for readability and maintaining.